server/pkg/thread/thread_test.go

```diff
@@ -8,14 +8,17 @@ import (
 
 func TestThread_ThreadType(t *testing.T) {
```

Comment:
```suggestion
func TestThread_Getters(t *testing.T) {
```

As I pointed out before.

`TestSTRUCTNAME_METHODNAME(t *testing.T)`

Reply:
Done!